src/Models/DomainVerification.php

```diff
@@ -3,8 +3,9 @@
 namespace SunAsterisk\DomainVerifier\Models;
 
 use Illuminate\Database\Eloquent\Model;
+use SunAsterisk\DomainVerifier\Contracts\Models\DomainVerifiableInterface;
 
-class DomainVerification extends Model
+class DomainVerification extends Model implements DomainVerifiableInterface
```

Comment:
It is the same issue in the pull #12. Please see [my comment](https://github.com/sun-asterisk-research/laravel-domain-verify/pull/12#discussion_r316944875).